examples/standby/standby.c

```diff
@@ -0,0 +1,71 @@
+// based on https://paste.sr.ht/blob/b9b4fb45cbc70f2db7e31a77a6ef7dd2a7f220fb
+// Could be defined here, or in the processor defines.
+#define SYSTEM_CORE_CLOCK 48000000
+
+#include "../../ch32v003fun/ch32v003fun.h"
```

Comment:
```suggestion
#include "ch32v003fun.h"
```

Reply:
my LSP (clangd) doesn't read the include paths in the makefiles, hence the relative path.
Besides being ugly, is it bad?

Reply:
The ugliness and inconsistency with the rest of the project is rough.  Please find a way of making your dev environment include that include path.